data/Language/Pascal.yaml

```diff
@@ -72,8 +72,8 @@ keywords:
   - with
   - xor
 datatypes:
-  - string
-  - boolean
-  - integer
-  - real
-  - char
+  - boolean_upper_case
+  - char_pascal
+  - integer_pascal
+  - real_pascal
+  - string_pascal
```

Comment:
no new line character at the end of file (new-line-at-end-of-file)

*Origin: YAMLLintBear, Section: `yaml`.*

Reply:
Hmm